### ISSUE TITLE
Autocomplete in :refer should only suggest symbols in that namespace

### ISSUE THREAD

**Issue by danielcompton** [cursiveclojure/cursive#852]
When I'm `:refer`ing a symbol into a namespace, Cursive's autocomplete suggests symbols from many namespaces. Could you restrict the autocomplete to only suggest symbols in the namespace I'm referring?

**Comment by cursiveclojure** [comment]
Yes, this is planned, see [here](https://cursiveclojure.com/archive/1153.html).
